dbtmetabase/utils.py

```diff
@@ -16,9 +15,19 @@ def get_version() -> str:
     except ModuleNotFoundError:
         logging.debug("No _version.py found")
 
+    # importlib is only available on Python 3.6+
```

Comment:
You can do this instead of catching for an error https://github.com/gouline/dbt-metabase/blob/master/setup.py#L6

Reply:
Isn't "Ask for forgiveness, not permission" a bit more Pythonic? But I agree that the nested try-except here was not looking that great and now it's a bit cleaner 😄 .